store/datastore/utils.go

```diff
@@ -38,34 +39,33 @@ func rebind(query string) string {
 
 // helper function that converts a simple repsitory list
 // to a sql IN statment.
-func toList(listof []*model.RepoLite) (string, []interface{}) {
-	var size = len(listof)
-	if size > 999 {
-		size = 999
-		listof = listof[:999]
-	}
-	var qs = make([]string, size, size)
-	var in = make([]interface{}, size, size)
-	for i, repo := range listof {
-		qs[i] = "?"
-		in[i] = repo.FullName
-	}
-	return strings.Join(qs, ","), in
-}
+func toList(listof []*model.RepoLite) ([]string, [][]interface{}) {
+	var total = len(listof)
```

Comment:
can we avoid returning a `[][]interface{}` and instead do something like:

```
const limit = 999

total := len(listof)
if total == 0 {
    return feed
}

pages := total / limit
if total % limit != 0 {
  pages++
}

for i:=0;i<pages;i++ {
    stmt, args := toList(listof, i*limit, limit)

    var tmpFeed []*model.Feed
    err = meddler.QueryAll(db, &tmpFeed, 
      fmt.Sprintf(userFeedQuery, stmt), args...)
    if err != nil {
      return nil, err
    }

    feed = append(feed, tmpFeed...)
}

if len(feed) <= 50 {
   return feed, nil
}

// sort

return feed[:50], nil
```

where `start` and `limit` indicate where to slice

```
func toList(listof []*model.RepoLite, start, limit int) ([]string, []interface{}) {
   if len(listof) != limit {
     listof = listof[start,limit]
   }
```

it could make sense to slice the `listof` before invoking `toList` as well ... I don't have big preference either way